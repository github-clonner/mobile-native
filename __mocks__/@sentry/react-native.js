export const captureMessage = jest.fn();
export default {};
