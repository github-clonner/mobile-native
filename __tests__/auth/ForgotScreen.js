import 'react-native';
import React from 'react';

import ForgotScreen from '../../src/auth/ForgotScreen';

jest.mock('../../src/auth/AuthService');

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('ForgotScreen component', () => {
  it('should renders correctly', () => {
    const navigation = {};
    const route = { params: {} };
    const forgotScreen = renderer
      .create(<ForgotScreen navigation={navigation} route={route} />)
      .toJSON();
    expect(forgotScreen).toMatchSnapshot();
  });
});
