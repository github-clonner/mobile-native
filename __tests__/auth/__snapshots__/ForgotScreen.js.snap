// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotScreen component should renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Array [
        undefined,
        undefined,
      ],
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        undefined,
        undefined,
      ]
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
    >
      <Text
        style={
          Array [
            undefined,
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        Forgot your password?
      </Text>
      <Text
        style={
          Array [
            undefined,
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        To request a new password, enter your username
      </Text>
      <View>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#AEB0B8",
                    "fontFamily": "Roboto",
                    "fontSize": 16,
                    "fontWeight": "600",
                    "marginBottom": 5,
                  },
                  undefined,
                ]
              }
            >
              Username
            </Text>
          </View>
        </View>
        <TextInput
          allowFontScaling={true}
          autoCapitalize="none"
          editable={false}
          onChangeText={[Function]}
          placeholder=""
          placeholderTextColor="#444"
          rejectResponderTermination={true}
          returnKeyType="done"
          style={
            Array [
              undefined,
              undefined,
            ]
          }
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
      <View
        style={
          Array [
            undefined,
            undefined,
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#5DBAC0",
              "borderRadius": 36,
              "opacity": 1,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "white",
                },
                undefined,
              ]
            }
          >
             
            Go back
             
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#5DBAC0",
              "borderRadius": 36,
              "opacity": 1,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "white",
                },
                undefined,
              ]
            }
          >
             
            Continue
             
          </Text>
        </View>
      </View>
    </RCTSafeAreaView>
  </View>
</View>
`;
